documentation/specs/src/economics/public-goods-funding.md

```diff
@@ -198,44 +200,36 @@ impl addRecipient for cPgfRecipients
 impl remRecipient for cPgfRecipients
 ```
 
-### End of Term Summary
+## PGF stewards incentives
 
-At the end of each term, the council is encouraged to submit a "summary"  which describes the funding decisions the councils have made and their reasoning for these decisions. This summary will act as an assessment of the council and will be the primary document on the basis of which governance should decide whether to re-elect the council.
+Being a PGF steward is (should be) hard work. Stewards must invest their time in crafting proposals, being in touch with the opinions of the Namada community, and taking the risks associated with being voted out. In order to incentivise this effort, Namada allocates a set amount of NAM inflation that is directed towards the Stewards. This parameter is set by governance and is included in genesis by default. That allocation is then split equally between each PGF Steward.
 
 ## Addresses
 Governance adds 1 internal address:
 
 `PGF` internal address
 
-The internal address VP will hold the allowance the 10% inflation of NAM. This will be added in addition to what was unspent by the previous council. It is important to note that it is this internal address which holds the funds, rather than the PGF council multisig.
-
-The council should be able to burn funds (up to their spending cap), but this hopefully should not require additional functionality beyond what currently exists.
+The internal address VP will hold the allowance the 10% inflation of NAM. This funding will be allocated to the internal address at the start of each epoch. It is important to note that it is this internal address which holds the funds, rather than any of the Stewards' multisigs.
```

Comment:
```suggestion
The internal address VP will hold the allowance the 10% inflation of NAM. This funding will be allocated to the internal address at the start of each epoch. It is important to note that it is this internal address which holds the funds, rather than any of the stewards' multisigs.
```